lib/ipmi_sel.c

```diff
@@ -1244,6 +1244,9 @@ ipmi_get_oem_desc(struct ipmi_intf * intf, struct sel_event_record * rec)
 	case IPMI_OEM_SUPERMICRO_47488:
 		desc = get_supermicro_evt_desc(intf, rec);
 		break;
+	case IPMI_OEM_QUANTA:		
+		desc = get_quanta_evt_desc(intf, rec);
```

Comment:
You get `warning: implicit declaration of function ‘get_quanta_evt_desc’` here, and also (as a result) `warning: implicit declaration of function ‘get_quanta_evt_desc’`. Please fix.